release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -589,6 +589,13 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-5-known-issues"]
 == Known issues
 
+When upgrading to a new {product-title} z-stream release, connectivity to the API server might be interrupted as nodes are upgraded, causing API requests to fail.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1791162[*BZ#1791162*])
```

Comment:
There are + symbols preceding the link, not sure if that's intentional. 

Otherwise LGTM.